src/test/suite/settings.test.ts

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
Increasing the code coverage % by adding these new tests to some unrelated code to this change.